src/main/java/org/folio/service/export/storage/AWSStorageServiceImpl.java

```diff
@@ -106,10 +113,20 @@ public void storeFile(FileDefinition fileDefinition, String tenantId) {
         multipleFileUpload.waitForCompletion();
       } catch (InterruptedException e) {
         LOGGER.error(e.getMessage());
+        populateErrorLog("S3 connection issue", fileDefinition.getJobExecutionId(), tenantId);
         Thread.currentThread().interrupt();
       } finally {
         transferManager.shutdownNow();
       }
     }
   }
+
+  private void populateErrorLog(String reason, String jobExecutionId, String tenantId) {
```

Comment:
looks like this method will be used in all classes where we will populate error log, worth thinking to move it to ErrorLogService